src/App.tsx

```diff
@@ -43,16 +42,23 @@ export const App: React.FC<AppProps> = ({
     };
   }, []);
 
+  // TODO: Find better place for MainMenu items to be decided.
+  const menuItems: MenuButtonItem[] = [
+    { label: 'Search 🔎', element: <SearchBar />, action: (id) =>  {} },
+    { label: 'Exit 🛑', action: () => process.exit(1) },
+  ];
```

Comment:
Let's just remove the emojis for now. Maybe we'll figure out another solution later.